testing/bdd/features/handover/handover.feature

```diff
@@ -0,0 +1,84 @@
+Feature: Handover algorithm
+# when a document is received with a higher value ebuttp:authorsGroupControlToken than that most recently received in the
+# currently selected sequence the Handover Manager shall switch to that document's sequence, i.e. it shall emit 
+# a document in its output sequence corresponding to the received document with the new control token without delay.
+
+    Examples:
+    | xml_file            |   
+    | handover_algorithm.xml |   
+    
+
+  # SPEC-CONFORMANCE: R23, R25, 120
+  Scenario: Switch to sequence with higher value token
+    Given an xml file <xml_file>
+    And a handover node
+    When it has sequence identifier <seq_id_1>
+    And it has authorsGroupControlToken <token_1>
+    And the document is generated
+    And the document is processed 
+    # ^^^ ASSUMING THIS STEP COVERS CACHED TOKENS, IE THE SECOND DOCUMENTS DOESN'T HAVE TO FOLLOW IMMEDIATELY 
+    And another document arrives 
+    And it has sequence identifier <seq_id_2>
+    And it has authorsGroupControlToken <token_2>
+    And the document is generated
+    And the document is processed
+    Then the emitted document has <authorsGroupSelectedSequenceIdentifier> 
+
+    Examples:
+    | seq_id_1   | token_1         | seq_id_2   | token_2         | authorsGroupSelectedSequenceIdentifier |  
+    | sequence_a | 12345           | sequence_b | 12346           | sequence_b                             |  
+    | sequence_a | 1               | sequence_b | 0               | sequence_a                             |  
+    | sequence_a | 0               | sequence_b |                 | sequence_a                             |  
+    | sequence_a | 999999999999999 | sequence_b | 999999999999998 | sequence_a                             |  
```

Comment:
This 15 digit number > sys.maxint on 32 bit Python 2.7 so it will probably fail on some machines (it is platform dependent). We can reduce it to a smaller value, e.g. 9 digits.